src/ducks/future/EstimatedBudgetCard.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+import { Media, Bd, Img } from 'cozy-ui/transpiled/react/Media'
+import RightIcon from 'cozy-ui/transpiled/react/Icons/Right'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import Figure from 'cozy-ui/transpiled/react/Figure'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { queryConnect } from 'cozy-client'
+import { getCurrencySymbol } from 'utils/currencySymbol'
+
+import { recurrenceConn } from 'doctypes'
+import Elevated from 'components/Elevated'
+import useEstimatedBudget from './useEstimatedBudget'
+
+const DumbEstimatedBudgetCardTransactions = () => {
```

Comment:
is it really dumb since it uses `useEstimateBudget` ?

Reply:
Code not there anymore so not relevant, there is no dumb for FutureBalanceCard.